CONTRIBUTING.md

```diff
@@ -18,31 +19,56 @@ If your change is more than a simple fix, please don't just create a big pull re
+9. Update the [documentation](#update-documentation) with additional information as needed.
+
+## Update Documentation
+
+The Mixed Reality Toolkit provides the following forms of documentation.
+
+- API
+This documentation is generated from the product code and is reviewed as part of **all** pull requests.
+- Conceptual
+Conceptual documentation is hosted on https://docs.microsoft.com/en-us/windows/mixed-reality. Please submit our changes via Pull Requuest at https://github.com/MicrosoftDocs/mixed-reality.
+- Readme.md files
+As part of your pull request, please update (or create) the Readme.md file in the appropriate feature folder. This will allow GitHub users to gain a high-level understanding of your new feature.
+
+
+## Supported Platforms
+
+The Mixed Reality Toolkit supports thge following mixed reality (AR/VR/XR) platforms.
+
+- Windows Mixed Reality
+    - Immersive devices
+    - Microsoft Hololens
+- Open VR
 
 # Pull request
 
-Start a GitHub pull request to merge your topic branch into the [main repository's Dev_Working_Branch](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/Dev_Working_Branch). 
-(If you are a Microsoft employee and are not a member of the [Microsoft organization on GitHub](https://github.com/Microsoft) yet, please link your Microsoft and GitHub accounts on corpnet by visiting [Open Source at Microsoft](https://opensource.microsoft.com/) before you start your pull request. There's some process stuff you'll need to do ahead of time.)
+Start a GitHub pull request to merge your topic branch into the [mrtk_development](https://github.com/microsoft/mixedrealitytoolkit-unity/tree/mrtk_development) branch. 
+> If you are a Microsoft employee and are not a member of the [Microsoft organization on GitHub](https://github.com/Microsoft), please link your Microsoft and GitHub accounts on corpnet by visiting [Open Source at Microsoft](https://opensource.microsoft.com/) before you start your pull request. There's some process stuff you'll need to do ahead of time.
+
 If you haven't contributed to a Microsoft project before, you may be asked to sign a [contribution license agreement](https://cla.microsoft.com/). 
 A comment in the PR will let you know if you do.
 
 The project maintainers will review your changes. We aim to review all changes within three business days.
-Address any review comments, force push to your topic branch, and post a comment letting us know that there's new stuff to review.
+Address any review comments, push to your topic branch, and post a comment letting us know that there's new stuff to review.
 
 # Merge
 
 If the pull request review goes well, a project maintainer will merge your changes. Thank you for helping improve MixedRealityToolkit!
+
```

Comment:
Empty spaces?

Reply:
?

Reply:
not sure where they came from... removing